passes/opt/opt_merge.cc

```diff
@@ -268,13 +267,13 @@ struct OptMergeWorker
 			}
 
 			did_something = false;
-			dict<std::string, RTLIL::Cell*> sharemap;
+                        std::unordered_map<FingerPrint, RTLIL::Cell*> sharemap;
```

Comment:
I think this should remain `dict`; the Yosys codebase tends to avoid STL associative containers due to them having subtly different behaviour across platforms.

Reply:
Done.